modules/homebrew.nix

```diff
@@ -204,8 +212,8 @@ in
     system.activationScripts.homebrew.text = mkIf cfg.enable ''
       # Homebrew Bundle
       echo >&2 "Homebrew bundle..."
-      if [ -f /usr/local/bin/brew ]; then
-        PATH=/usr/local/bin:$PATH ${brew-bundle-command}
+      if [ -f ${cfg.brewPath} ]; then
+        PATH=''$(dirname ${cfg.brewPath}):$PATH ${brew-bundle-command}
```

Comment:
Since the dirname is needed here I'd prefer a brewPrefix or package option instead of the full path to the binary.

Reply:
done